apps/entropy-tester/src/index.ts

```diff
@@ -0,0 +1,168 @@
+  const configs = YAML.parse(fs.readFileSync(configPath, "utf-8"));
+  const loadedConfigs = [];
+  for (const config of configs) {
+    const interval = timeToSeconds(config["interval"]);
+    const contracts = Object.values(DefaultStore.entropy_contracts).filter(
+      (contract) => contract.chain.getId() == config["chain-id"],
+    );
+    if (contracts.length === 0) {
+      throw new Error(
+        `Can not find the contract for chain ${config["chain-id"]}, check contract manager store.`,
+      );
+    }
+    if (contracts.length > 1) {
+      throw new Error(
+        `Multiple contracts found for chain ${config["chain-id"]}, check contract manager store.`,
+      );
+    }
+    loadedConfigs.push({ contract: contracts[0], interval });
+  }
+  return loadedConfigs;
+}
+
+async function testLatency(
+  contract: EvmEntropyContract,
+  privateKey: PrivateKey,
+  logger: Logger,
+) {
+  const provider = await contract.getDefaultProvider();
+  const userRandomNumber = contract.generateUserRandomNumber();
+  const requestResponse = await contract.requestRandomness(
+    userRandomNumber,
+    provider,
+    privateKey,
+    true, // with callback
+  );
+  // Read the sequence number for the request from the transaction events.
+  const sequenceNumber = parseInt(
+    requestResponse.events.RequestedWithCallback.returnValues.sequenceNumber,
+  );
+  logger.info(
+    { sequenceNumber, txHash: requestResponse.transactionHash },
+    `Request submitted`,
+  );
+
+  const startTime = Date.now();
+
+  // eslint-disable-next-line no-constant-condition
+  while (true) {
+    await new Promise((resolve) => setTimeout(resolve, 2000));
+    const request = await contract.getRequest(provider, sequenceNumber);
+    logger.debug(request);
+
+    if (parseInt(request.sequenceNumber) === 0) {
+      // 0 means the request is cleared
+      const endTime = Date.now();
+      logger.info(
+        { sequenceNumber, latency: endTime - startTime },
+        `Successful callback`,
+      );
+      break;
+    }
+    if (Date.now() - startTime > 60000) {
+      logger.error(
+        { sequenceNumber },
+        "Timeout: 60s passed without the callback being called",
+      );
+      break;
+    }
+  }
+}
+
+yargs(hideBin(process.argv))
+  .parserConfiguration({
+    "parse-numbers": false,
+  })
+  .command({
+    command: "run",
+    describe: "run the tester until manually stopped",
+    builder: {
+      validate: {
+        description: "Only validate the configs and exit",
+        type: "boolean",
+        default: false,
+        required: false,
+      },
+      config: {
+        description: "Yaml config file",
+        type: "string",
+        required: true,
+      },
+      "private-key": {
+        type: "string",
+        required: true,
+        description:
+          "Path to the private key to sign the transactions with. Should be hex encoded",
+      },
+    },
+    handler: async (argv: any) => {
```

Comment:
We should never use `any`; yargs will properly typecheck the arguments so the annotation shouldn't even be needed and adding it just cripples the type system.  `any` is a cancer and it will spread and cause the type checker to silently stop verifying a lot more of your code than you probably intend.

If you must, use `unknown`, but never use `any`.